docs/source/start/declarative.rst

```diff
@@ -84,8 +84,17 @@ our test model:
 
    - parent: !uuid f28ec0f8-f3b3-43a0-8af7-79f194b29a2d
 
-Creating objects
-----------------
+Extending objects
+-----------------
+
+The following subsections show how to create completely new, reference and move
+objects using examples of declarative YAML files on
```

Comment:
I'd rewrite this sentence as "[...] create completely new objects, or reference and move already existing ones [...]". Reads a lot smoother to me.